util/cmake/FindSeqAn.cmake

```diff
@@ -174,6 +171,8 @@ endif ()
 # Require C++14
 # ----------------------------------------------------------------------------
 
+# The standard library of visual studio defines __cplusplus still as
```

Comment:
It is unrelated to thr STDLIB, check for COMPILER_MSVC and COMPILER_WINTEL (?) instead and make the comment explain that its about compiler, not stdlib ;)